src/subdomains/supporting/support-issue/services/support-issue.service.ts

```diff
@@ -30,25 +30,26 @@ export class SupportIssueService {
     private readonly supportIssueNotificationService: SupportIssueNotificationService,
   ) {}
 
-  async createTransactionIssue(
-    userDataId: number,
-    transactionId: number,
-    dto: CreateTransactionIssueDto,
-  ): Promise<void> {
+  async createIssue(userDataId: number, dto: CreateSupportIssueDto): Promise<void> {
+    let entity = await this.create(userDataId, SupportIssueType.GENERIC_ISSUE, dto);
```

Comment:
Maybe a existing check? With userDataId and reason at least? Maybe max. 2-3 Tickets with same reason for one userData or something like that, else spamming is easy 😅